kratos/utilities/element_size_calculator.cpp

```diff
@@ -867,7 +867,7 @@ double ElementSizeCalculator<2,4>::AverageElementSizeDerivative(
 template<>
 double ElementSizeCalculator<2,9>::AverageElementSize(const Geometry<Node >& rGeometry)
 {
-    return rGeometry.Length();
+    return rGeometry.Length() * 2.0;
```

Comment:
hm so thats a hack for the hack?
Can you add a comment? :D

Reply:
done :)